pkgs/cupertino_http/macos/cupertino_http.podspec

```diff
@@ -0,0 +1,28 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint cupertino_http.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'cupertino_http'
+  s.version          = '0.0.1'
+  s.summary          = 'Flutter Foundation URL Loading System'
+  s.description      = <<-DESC
+  A Flutter plugin for accessing the Foundation URL Loading System.
+                       DESC
+  s.homepage         = 'https://github.com/dart-lang/http/tree/master/pkgs/cupertino_http'
+  s.license          = { :type => 'BSD', :file => '../LICENSE' }
+  s.author           = { 'TODO' => 'use-valid-author' }
```

Comment:
author?

Reply:
I'll fix this when Kevin gives me a group name.